packages/block-library/src/navigation/index.php

```diff
@@ -1354,8 +1354,8 @@ function block_core_navigation_get_most_recently_published_navigation() {
  * children, the `wp_navigation` post's `_wp_ignored_hooked_blocks` meta is checked to see if any
  * of those hooked blocks should be exempted from insertion.
  *
- * @param array   $inner_blocks Parsed inner blocks of a Navigation block.
- * @param WP_Post $post         `wp_navigation` post object corresponding to the block.
+ * @param array        $inner_blocks Parsed inner blocks of a Navigation block.
+ * @param WP_Post|null $post         Optional. `wp_navigation` post object corresponding to the block. Default null.
```

Comment:
I briefly tried to glance over the callsites for this function. Is it possible that we eliminated all cases where `$post` could've been omitted? (If yes, we might want to make the argument mandatory.)

Reply:
Good spot. I've updated here https://github.com/WordPress/gutenberg/pull/58378/commits/ecd6c3f73563402694d1ba50c7bb3c2ab1d96864